src/App/Commands/DbImportCommand.php

```diff
@@ -23,27 +24,32 @@ protected function configure()
         $this->setName('db-import')
             ->setDescription('Executes db import command')
             ->setHelp('Import given db into default database.')
-            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import');
+            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import')
+            ->addOption('dbname', NULL, InputOption::VALUE_OPTIONAL, 'The database in that we want to import the data');
```

Comment:
The database that you want to import the data in

Reply:
Thx